internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -174,3 +174,19 @@ func (p Plan) GetSizeID() (string, error) {
 	}
 	return t[1], nil
 }
+
+type Quantity string
+
+func (q Quantity) String() string {
```

Comment:
Method declaration in `go` is just syntactic sugar: your method will be translated to:
```go
func String(q Quantity) string {
```

thus, everytime you call `String` your string gets copied.

That would be solved by using a pointer receiver:
```suggestion
func (q *Quantity) String() string {
```

Reply:
Thanks @ziccardi. I'll update